### PR TITLE
Fuck tape - Silicons can now go under tape

### PR DESCRIPTION
**Pull request by JamieD1**
Fuck tape

Silicons can now go under tape

:cl:  
rscadd: Silicons can now go under tape
/:cl:

### CODE REVIEW

File: code/game/objects/items/barriertape.dm [hunk 1]
```diff
@@ -121,6 +121,8 @@
 		return
 	if(mover.pass_flags & (PASSGLASS|PASSTABLE|PASSGRILLE))
 		return 1
+	if(issilicon(mover))
+		return 1
```
Comment (adamsong) [inline code/game/objects/items/barriertape.dm]:
```suggestion
		return TRUE
```

**Review by adamsong — CHANGES_REQUESTED** [review]

**Comment by maxion12345** [comment]
combat mechs cannot pass tape either and have to smash it :(

**Comment by stwordcitizen** [comment]
Jamie engineering tape is currently the best way to get away from someone bar an instant rcd, maybe make tape have a limited number of uses?

**Comment by JamieD1** [comment]
> Jamie engineering tape is currently the best way to get away from someone bar an instant rcd, maybe make tape have a limited number of uses?

Yeah Ill do some nerfs later today.

**Pull request merged**